packages/installer/cos.sh

```diff
@@ -4,17 +4,16 @@ set -e
 PROG=$0
 
 ## Installer
-PROGS="dd curl mkfs.ext4 mkfs.vfat fatlabel parted partprobe grub2-install grub2-editenv"
-DISTRO=/run/rootfsbase
-ISOMNT=/run/initramfs/live
-ISOBOOT=${ISOMNT}/boot
-TARGET=/run/cos/target
-RECOVERYDIR=/run/cos/recovery
-RECOVERYSQUASHFS=${ISOMNT}/recovery.squashfs
-GRUBCONF=/etc/cos/grub.cfg
+_PROGS="dd curl mkfs.ext4 mkfs.vfat fatlabel parted partprobe grub2-install grub2-editenv"
+_DISTRO=/run/rootfsbase
+_ISOMNT=/run/initramfs/live
+_TARGET=/run/cos/target
+_RECOVERYDIR=/run/cos/recovery
+_RECOVERYSQUASHFS=${_ISOMNT}/recovery.squashfs
+_GRUBCONF=/etc/cos/grub.cfg
 
 # Default size (in MB) of disk image files (.img) created during upgrades
-DEFAULT_IMAGE_SIZE=3240
+_DEFAULT_IMAGE_SIZE=3240
```

Comment:
This one is intended to be tweakable too

Reply:
should be fixed by the change below